laws/src/main/scala/cats/laws/NonEmptyParallelLaws.scala

```diff
@@ -0,0 +1,21 @@
+package cats
+package laws
+
+
+/**
+  * Laws that must be obeyed by any `cats.NonEmptyParallel`.
+  */
+trait NonEmptyParallelLaws[M[_], F[_]] {
+  def P: NonEmptyParallel[M, F]
+
```

Comment:
Should we say that the Functors should be the same. If not it seems pretty hard to reason about how they are related.

Reply:
Yes that's a good idea, I'll add a law!